### ISSUE TITLE
Search Issues

### ISSUE THREAD

**Issue by jamiefolsom** [performant-software/DM#43]
New issue: strings are not accurately being searched. In other words, a search string doesn't always return results which contain that string (Discussed with @gremid in May).

Search feature issue - search returns text items when some characters are searched for (like “_” or “_****”) that do not contain these characters in question. Conversely, searching for character strings like  “[ ]” or “[“ does not return annotations in actuality do contain this string in their text.

Example: "????" as a query.

Also, partial strings are not matched -- might/might not be in scope for 1.0.0.

Options include:
- User Documentation, including how the syntax can be used.
- Change the parser, to make the search behavior more intuitive.

**Comment by SteveMarvin** [comment]
The story doesn't really specify what is to be done so much as what **might** be done, nor do any comments specify what **has** been done. Here's what I see:
* Searching for "josc" does not return results for items definitely present in teh text, such as "Joscelyn"
* Items so common that they must be present, such as "the", do not give any returns, saying "**No search results for “the”**" rather than "Too many results to display" or something like that.
* "????" and "****" as search terms return no results.
* "*" and "?" cannot be used as wildcards

In short, I can't tell what this story wanted to do, and I can't figure out what has been done. Can we reword the story or offer some comments as to how it was implemented?

Mind you, it may be that the purpose of this story was to make sure that all of those options in fact do nothing...I'm only rejecting this story to flag it as needing clarity.
